test/unit/modules/request-browser-spec.js

```diff
@@ -86,6 +86,151 @@ describe('Browser Request methods', () => {
         });
     });
 
+    const newConfig = {...helpers.client, roundValues: true};
+    const customClient = new KeenClient(newConfig);
+    const dummyNormalResponseUp = { result: 14.612}
+    const dummyRoundResponseUp = { result: 15 }
+
+    it('rounding values up', async () => {
+      fetch.mockResponseOnce(JSON.stringify(dummyNormalResponseUp));
+      const queryParams = {
+        analysis_type: 'average',
+        event_collection: 'pageviews',
+        timeframe: 'this_3_months'
+      };
+      await customClient
+        .query(queryParams)
+        .then(res => {
+          expect(res.result).toBe(dummyRoundResponseUp.result);
+        });
+    });
+
+    const dummyNormalResponseDown = { result: 14.212}
+    const dummyRoundResponseDown = { result: 14 }
+
+    it('rounding values down', async () => {
+      fetch.mockResponseOnce(JSON.stringify(dummyNormalResponseDown));
+      const queryParams = {
+        analysis_type: 'average',
+        event_collection: 'pageviews',
+        timeframe: 'this_3_months'
+      };
+      await customClient
+        .query(queryParams)
+        .then(res => {
+          expect(res.result).toBe(dummyRoundResponseDown.result);
+        });
+    });
+
+    const dummyNormalResponseGroupBy = {result: [
+      { result: 5.5 },
+      { result: 2.4 },
+      { result: 20.8 },
+      { result: 1.91 },
+    ]}
+    const dummyRoundResponseGroupBy = [
+      { result: 6 },
+      { result: 2 },
+      { result: 21 },
+      { result: 2 },
+    ]
+
```

Comment:
just like the comment above - let's encapsulate vars inside the IT function whenever possible